install/prereqs/debian.rst

```diff
@@ -25,8 +33,25 @@ If you get an error run:
 
    apt-get update --fix-missing
 
-and try again.
+and run again the app-get install above.
 
 .. note::
 
-   **New since 11.0**: The minimum gcc version required to compile the LSST Science Pipelines is **gcc 4.8.**
+   **New since 15.0**: The gcc compiler to be used shall support **C++ 14**. The gcc version used in ubuntu 16 to compile the LSST Science Pipelines is **gcc 5.4.**.
+
+..   It has been verified that the release **15.0** will not compile on a **Ubuntu 14** standard installation, due to gcc compiler too old.
+
+Additional installation steps for Ubuntu 14
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the case you are deploying the lsst_distrib in an ubuntu 14 machine, the gcc compilerinstalled will be the **4.8**. In order to get the gcc version **5.4** following steps have to be completed in addition at the above ones.
+
+.. code-block:: bash
+
+   sudo apt-get update
+   sudo apt-get install build-essential software-properties-common -y
+   sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+   sudo apt-get update
+   sudo apt-get install gcc-5 g++-5
+   sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-5 60 --slave /usr/bin/g++ g++ /usr/bin/g++-5
```

Comment:
Maybe add a warning that this is modifying system-wide configuration and will impact any other users on the machine — it's not equivalent to simply activating a devtoolset.